themes/bulma/assets/index.css

```
@@ -0,0 +1,407 @@
+/* ────────────────────────────────────────────────────────────────────────────
+ * @vuecs/elements Badge — outline variant.
+ * Bulma 1.0 has no `.tag.is-outlined` in core. The variant matrix tags
+ * outline cells with the marker classes `vc-badge-outline` +
+ * `vc-badge-outline-<color>`; this block paints them via design-system
+ * tokens. Borders use `-600` (matches solid bg saturation), text uses
+ * `-700` for emphasis.
+ * ──────────────────────────────────────────────────────────────────────── */
+
+.vc-badge-outline {
+    background-color: transparent;
+    border: 1px solid currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)